include/dlpack/dlpack.h

```diff
@@ -143,9 +143,16 @@ typedef struct {
  */
 typedef struct {
   /*!
-   * \brief The opaque data pointer points to the allocated data. This will be
-   * CUDA device pointer or cl_mem handle in OpenCL. This pointer is always
-   * aligned to 256 bytes as in CUDA.
+   * \brief The data pointer points to the allocated data. This will be CUDA
+   * device pointer or cl_mem handle in OpenCL. It may be opaque on some device
+   * types. This pointer is always aligned to 256 bytes as in CUDA. The
+   * `byte_offset` field should be used to point to the beginning of the data.
+   *
+   * Note that as of Nov 2021, multiply libraries (CuPy, PyTorch, TensorFlow,
+   * TVM, perhaps others) do not adhere to this 256 byte aligment requirement
```

Comment:
TVM does adhere to the alignment requirement in allocation

Reply:
Perhaps I misread the code then. Looking at the whole file I linked in the PR description, there is a function that ensures 64-byte alignment, but not 256-byte as far as I can tell: https://github.com/apache/tvm/blob/43c2ea72bcd9968531441189e172372c1d8a64cb/src/contrib/tf_op/tvm_dso_op_kernels.cc#L129. I may be looking in the wrong place - scouring multiple large unknown code bases was a bit of a pain.

Reply:
You get me :) Correct, I will merge it in as it is.

Reply:
Thanks!

I did some more browsing in TVM, there are allocators for CUDA/ROCm which do use 256 byte ([AllocDataSpace](https://github.com/apache/tvm/blob/56c4d07ac54d6b96d94f18031ac8ab2aa263d132/src/runtime/cuda/cuda_device_api.cc#L112)) indeed, but I came up empty beyond that.